community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/PhysicalLogCommandReaderV3_0_2.java

```diff
@@ -82,6 +83,9 @@ protected Command read( byte commandType, ReadableChannel channel ) throws IOExc
             return visitRelationshipTypeTokenCommand( channel );
         case NeoCommandType.LABEL_KEY_COMMAND:
             return visitLabelTokenCommand( channel );
+            // TODO do older physical logs need to support time zone tokens?
```

Comment:
No and this addition should bump the tx log format, see `org.neo4j.kernel.impl.storageengine.impl.recordstorage.RecordStorageCommandReaderFactory`. Basically don't add this time zone reading capability here, instead copy this class and add it to the new version instead. See outline of how to do that in `org.neo4j.kernel.impl.transaction.log.entry.LogEntryVersion` class javadoc, which I hope is up2date.

Reply:
Oki doki.